sriov/tests/conftest.py

```diff
@@ -15,4 +15,28 @@ def dut(settings):
     host = settings.config["dut"]["host"]
     user = settings.config["dut"]["username"]
     password = settings.config["dut"]["password"]
-    return ShellHandler(host, user, password)
\ No newline at end of file
+    return ShellHandler(host, user, password)
+
+@pytest.fixture(autouse=True)
+def initialization(dut, settings, pf_net_path):
+    reset_command(dut, settings, pf_net_path)
+
+@pytest.fixture()
+def pf_name(settings):
+    pf_name = settings.config["dut"]["interface"]["pf1"]["name"]
+    return pf_name
+
+@pytest.fixture()
+def pf_net_path(settings, pf_name):
+    pf_net_path = "/sys/class/net/"+ pf_name + '/device'
+    return pf_net_path
+
+@pytest.fixture
+def reset_vfs(dut, settings, pf_net_path):
+    yield
```

Comment:
maybe we can have a single clean up function which cover both the pre-test initialization and post-test clean up, something like:
@pytest.fixture(autouse=True)
def common_cleanup(dut, settings, pf_net_path):
     reset_command(dut, settings, pf_net_path)
     yield
     du.stop_testpmd()
      reset_command(dut, settings, pf_net_path)